### ISSUE TITLE
Running it?

### ISSUE THREAD

**Issue by fizixes** [Saurabhkr8038/Expert-Crypto-computing-machine#1]
On surface, this seems like a great idea but just ended up being issues after issue. I created the .env file which went fine. I then had to create a copy of the template folder for run.py. Looking at the code, run.py doesn't really do anything? It just displays the dashboard, but isn't nearly as robust as the description claims.

Then in scheduler.py:

from data_loader import fetch_data_from_ccxt
fetch_data_from_ccxt()
  
I had to change these lines to:

from data_loader import fetch_ohlcv
fetch_ohlcv()

Because the function fetch_data_from_ccxt just doesn't exist.
I then realized that fetching OHLC doesn't save it or anything, so was kind of a useless change unless I'm going to also schedule it also processing and saving the data...
Similar with the retrain model function

from model import CryptoModelTrainer
model = CryptoModelTrainer()
model.train_xgboost()

Also run.py never calls the scheduler module so they have be run separately (I Think?)
I'm sure this works great when implemented properly, or maybe I am just running it wrong. 
Please let me know if this is just half-baked or if its user error. I am willing to contribute.
Anaconda Environment Python 12.9  with only requirements.txt installed.
Thanks!

**Comment by Saurabhkr8038** [comment]
this is the prototype i had developed for a client the original project is getting developed right now and will be updated here once delivered to the client "However this repo is still a goldmine for someone who wants to build it, it contains more than enough details and all the code data " please feel free to contribute to this one though im more than happy if u are really interested.

**Comment by fizixes** [comment]
Ah okay, that makes a lot of since. I was not trying to poo poo the project, the framework is there and seems great. Have you made any updates you'd be willing to commit since uploading? I don't want to be working on the same issues and running over each other. Thanks!

**Comment by Saurabhkr8038** [comment]
i will update as soon as my work is done
